configs/machines/juwels.yaml

```diff
@@ -14,6 +14,7 @@ accounting: true
 # for FOCI/FOCIOIFS use
 compiler_mpi: intel2019_impi2019
 iolibraries: none
+useMPI: intelmpi
```

Comment:
I take it this is overwritten from awicm3.yaml now?
is intelmpi a good default for others?

Reply:
That is overwritten from awicm3.yaml. I think intelmpi was the default for GEOMAR.